src/lib/system-prompt.ts

```diff
@@ -10,7 +10,8 @@ I follow these rules when responding:
 - Format ALL lines of code to 80 characters or fewer
 - Use Mermaid diagrams when discussing visual topics
 - If using functions, only use the specific functions I have been provided with
-- If responding with math, use KaTeX syntax in your Markdown by enclosing it with double-dollar signs rather than parentheses, for example: $$O(n\\log n)$$
+- Inline or otherwise, I ALWAYS use KaTeX syntax in math Markdown by enclosing EVERY mathematical expression, equation, variable, and formula with double-dollar signs ($$), for example: $$O(n\\log n)$$, $$1024 * 1024 = 1048576$$, $$1024^2$$, $$X$$
```

Comment:
Looks good me, I found the system prompt will be impacted, maybe we can use `` here

```suggestion
- Inline or otherwise, I ALWAYS use KaTeX syntax in math Markdown by enclosing EVERY mathematical expression, equation, variable, and formula with double-dollar signs `$$`, for example: $$O(n\\log n)$$, $$1024 * 1024 = 1048576$$, $$1024^2$$, $$X$$
```

Reply:
Oh my bad this will break the code 😢

Reply:
Maybe using subitem ?
```suggestion
- Inline or otherwise, I ALWAYS use KaTeX syntax in math Markdown by enclosing EVERY mathematical expression, equation, variable, and formula with double-dollar signs ($$), for example: 
  - $$O(n\\log n)$$, $$1024 * 1024 = 1048576$$, $$1024^2$$, $$X$$
```

Reply:
I find 
```
\`($$)\`
```

seems to work